### DISCUSSION TITLE
[Feature Request] Give option to use local GlitchTip instead of Sentry

### DISCUSSION THREAD

**Discussion by dekoza** [fastapi/full-stack-fastapi-template#2137] [Questions] [answered]
[GlitchTip](https://glitchtip.com/) is an open-source Sentry alternative. It can be run on the same docker stack as the rest of services. It even uses the same API so using it is just a matter of adding its image to docker-compose and providing DSN.

**Comment by vanschelven** [comment, marked as answer]
The exact same thing could be said for [Bugsink](https://github.com/bugsink/bugsink/).

On the flip-side of that: since it's "just a DSN", might as well simply let people put in that DSN instead.
